presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -597,7 +597,7 @@ public boolean isHiveViewsEnabled()
         return hiveViewsEnabled;
     }
 
-    @Config("hive.views-execution.enabled")
+    @Config("hive.view-execution.enabled")
```

Comment:
or maybe even `hive.hive-view...` ? to indicate it's _hive_ connector's property for _hive views_

Reply:
Maybe we should call this `translate-hive-views`?